tests/AirtableClientKernel.php

```diff
@@ -25,9 +30,13 @@ public function registerBundles(): array
 
     protected function configureContainer(ContainerConfigurator $container, LoaderInterface $loader): void
     {
+        // Load test purpose configuration
         $container->extension('airtable_client', [
-            'key' => 'key',
-            'id' => 'id',
+            'key' => self::KEY,
+            'id' => self::ID,
         ]);
+
+        // Replace HttpClientInterface by a test-friendly HttpClient MockHttpClient
+        $container->services()->set(HttpClientInterface::class, MockHttpClient::class);
```

Comment:
The `MockHttpClient` is a very good tool to test our client HTTP as it exposes some methods.
When I do `->set(HttpClientInterface::class, MockHttpClient::class);` I tell to symfony replace the service that match `HttpClientInterface` with `MockHttpClient` so when I call `request` Symfony will use `MockHttpClient` and I can access to some helpers to check the options for the test